doc/api/net.md

```diff
@@ -895,6 +895,9 @@ added: v0.1.90
 Sets the socket to timeout after `timeout` milliseconds of inactivity on
 the socket. By default `net.Socket` do not have a timeout.
 
+Prior to Node.js 13.0.0, [`http.Server`][], [`https.Server`][] and
+[`http2.Http2Server`][] have a different default socket timeout value.
```

Comment:
Can we incorporate this information into the [YAML matter](https://github.com/nodejs/node/pull/31379/files#diff-50a406361a2834dc8097003eac9b916dL887-L889) instead of having it here?

Reply:
> Can we incorporate this information into the [YAML matter](https://github.com/nodejs/node/pull/31379/files#diff-50a406361a2834dc8097003eac9b916dL887-L889) instead of having it here?

Oh, wait, no, not that YAML matter. What's the rationale for putting this in `net` instead of `http`, `https`, and `http2` docs?

Reply:
Apparently some users are referring to `net` docs when working with http.Server (see #31378); prior to Node.js 13, default values are consistent across all sockets in Node.js codebase EXCEPT for the timeout, I figured adding a note would help avoid the confusion.
Users who refer to `http`[s2] docs already have the correct information, I'm not sure there is much confusion from that side.